### PR TITLE
update icon: r (original, plain, line)

### PR DESCRIPTION
**Pull request by canaleal**
## Double check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened
- [x] PR name matches the format *new icon: <i>Icon name</i> (<i>versions separated by comma</i>)*. More details [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#overview)
- [x] PR's base is the `develop` branch.
- [x] Your icons are inside a folder as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#organizational-guidelines)
- [x] SVG matches the standards laid out [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#svgStandards)
- [x] A new object is added in the `devicon.json` file as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#-updating-the-deviconjson-)

**This PR is related to #1998**

## Link to prove your SVG is correct and up-to-date.

- [R Website](https://www.r-project.org/)
- [R Logo (from website)](https://www.r-project.org/logo/)

Optimized the SVGs and created a line SVG.
Updated the color to `#000000` as the old color was randomly picked from the original SVG.

### CODE REVIEW

File: icons/r/r-line.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...6h-.006zm.992.992h42.998s4.857.093 9.698 2.483c4.84 2.39 9.631 6.976 9.631 16.228 0 9.26-4.591 14.122-9.217 16.768-4.625 2.646-9.256 2.994-9.256 2.994l-.106.97c-.048-.016.952.288 2.182.716s2.737.997 3.812 1.54c.744.378 2.048 1.147 2.944 2.1l.002.002a7.822 7.822 0 0 1 1.325 1.903l.008.017 14.714 24.808-23.314.01-11.312-21.244-.006-.008s-.593-1.018-1.369-2.186c-.775-1.167-1.698-2.47-2.535-3.147-.606-.49-1.058-.844-1.554-1.085-.495-.242-1.022-.345-1.664-.345h-6.312l.003 28-20.672.01zM74.174 56.45v.496l-.006 12.657 13.619-.004h.002s1.624 0 3.261-.835c1.638-.836 3.309-2.6 3.309-5.847 0-3.305-1.68-5.027-3.326-5.79-1.647-.765-3.273-.67-3.273-.67zm.993.992 12.633.008h.014s1.411-.07 2.802.575 2.75 1.906 2.75 4.892c0 2.92-1.367 4.248-2.767 4.963-1.4.715-2.814.727-2.814.727l-12.624.004zm9.273 31.341-.545.052a101.25 101.25 0 0 1-8.011.458l-.487.01.002 10.602.558-.07a88.168 88.168 0 0 0 12.448-2.484l.597-.165-.789-1.482c-.673-1.256-1.336-2.52-2.02-3.78l-.003-.007-.004-.006a41.248 41.248 0 0 0-1.746-2.725zm-.758 1.058a40.18 40.18 0 0 1 1.64 2.556.695.695 0 0 1 .007.014c.676 1.245 1.334 2.505 2.008 3.762l.184.345a87.181 87.181 0 0 1-11.13 2.222l-.002-8.482c2.5-.058 4.937-.193 7.293-.417z"/></svg>
```
Comment (Snailedlt) [inline icons/r/r-line.svg:1]: Does it work without the `preserveAspectRatio="xMidYMid"` attrbitue here?

**Review by Snailedlt — COMMENTED** [review]
I'm not so sure about picking black as the main color, imo it should be one of the blues used in the main logo, or on their website. What do you think?

Preview:
image
image
image

File: devicon.json [hunk 2]
```diff
@@ -8155,14 +8155,15 @@
...
             ],
             "font": [
-                "original",
-                "plain"
+                "plain",
+                "line"
             ]
         },
-        "color": "#2369bc",
+        "color": "#000",
```
Comment (Snailedlt) [inline devicon.json:8166]: Perhaps we can use the color in-between the two blues in the color-gradient as the main color?
```suggestion
        "color": "#1f65b7",
```

Here's how I calculated the color: https://jsfiddle.net/5p2xjucv/26/

**Review by Snailedlt — COMMENTED** [review]
Is it just me or are the versions a bit larger than  the viewbox?
image
image

**Review by Snailedlt — APPROVED** [review]
LGTM! ✔️ 
Thank you for another great contribution! 💯 🙇
image
image
image

**Review by lunatic-fox — APPROVED** [review]
Great job! 👍🏼

**Pull request merged**